### ISSUE TITLE
WSA does not respect changes to android:resizeableActivity in AndroidManifest

### ISSUE THREAD

**Issue by ulysses-meza** [microsoft/WSA#334]
### Steps to reproduce

Write a .NET MAUI or Forms application that:
- Set `<application android:resizeableActivity="true">` in Platforms\Android\AndroidManifest.xml :

```xml
<application android:label="TestApp"  android:resizeableActivity="true">
...
</application
```
- Install the app via adb to WSA and note when you try to resize the app it allows resize

- Set `<application android:resizeableActivity="false">` in Platforms\Android\AndroidManifest.xml :

```xml
<application android:label="TestApp"  android:resizeableActivity="false">
...
</application
```
- Install the app via adb to WSA and note that it no longer supports resize (as expected)

- Set `<application android:resizeableActivity="true">` in Platforms\Android\AndroidManifest.xml :

```xml
<application android:label="TestApp"  android:resizeableActivity="true">
...
</application
```
- Install the app via adb to WSA

### ✔️ Expected Behavior

App should allow you to resize the window

### ❌ Actual Behavior

the app does not resize the window correctly and instead shows you the reload button and says that the app does not support resize
![Screenshot 2023-06-15 095852](https://github.com/microsoft/WSA/assets/136852672/eb1c7266-8e26-4244-bb49-355038f4d4a6)

### Please specify the version of Windows Subsystem for Android

2305.40000.5.0

**Comment by stranchand** [comment]
I'm facing the same issue.

It was mentionned [here](https://github.com/microsoft/WSA/discussions/322#discussioncomment-6223857) too

**Comment by nieubank** [comment]
Does the app specify an explicit orientation? Also what SDK target is specified?

**Comment by ulysses-meza** [comment]
Orientation is not specified explicitly we allow orientation change, and the targetsdkversion is 31

**Comment by stranchand** [comment]
For me, in a native Android app, android is set to `landscape`, with :
-  `compileSdkVersion 33`
- `minSdkVersion 22`
- `targetSdkVersion 33`

**Comment by nieubank** [comment]
Thanks for the details. We don't really do anything special in this space in WSA other than enabling the AOSP size compat mode feature. 

We have some heuristics for apps that declare resize support but don't handle it well, but those are only enabled for apps acquired from the store so that isn't in play here. 

It almost looks like a deployment package info caching issue. We'll have to investigate but this is looking like an Android bug that just isn't hit on most platforms. We'll keep you updated.

**Comment by stranchand** [comment]
> We have some heuristics for apps that declare resize support but don't handle it well, but those are only enabled for apps acquired from the store so that isn't in play here.

I'm not sure that I understand what you said here... (Sorry, i'm not a native speaking english)

Does this mean that if I test resize management changes in my app (to test what is describe [here](https://developer.amazon.com/apps-and-games/blogs/2023/02/resizing-apps-in-wsa) ) that i cannot see changes until I download my app from the Amazon Store (and so that by just installing my apk using adb I would not see any change on it?)

Thank's in advance for your reply.

**Comment by nieubank** [comment]
> > We have some heuristics for apps that declare resize support but don't handle it well, but those are only enabled for apps acquired from the store so that isn't in play here.
> 
> 
> 
> I'm not sure that I understand what you said here... (Sorry, i'm not a native speaking english)
> 
> 
> 
> Does this mean that if I test resize management changes in my app (to test what is describe [here](https://developer.amazon.com/apps-and-games/blogs/2023/02/resizing-apps-in-wsa) ) that i cannot see changes until I download my app from the Amazon Store (and so that by just installing my apk using adb I would not see any change on it?)
> 
> 
> 
> Thank's in advance for your reply.
> 
> 
> 
> 
> 
> 
> 
> 

I should have been more clear. Generally speaking if you're updating you're app is targeting an SDK level of 24+ you won't see any difference in behavior. The heuristics are essentially for abandonware. If you're following the guidance provided above, you're app will behave the same regardless of if it was acquired from the store or not.

**Comment by nieubank** [comment]
> Ok, thanks for your clarification :pray: 
> 
> 
> 
> So you confirm that if we declare in our manifest `android:resizeableActivity="true"`, resize should works and this "refresh floating action button" should not appear ? No matter explicit orientation, and without the need of handling `onConfigurationChanged` ?

Correct. I'll work on a blog post for how all this works since the existing documentation isn't as detailed as I'd like. Thanks for calling this out.

**Comment by Denisuu** [comment]
I have the same problem on my Onyx Leaf 2.

Is there a patch so this pop-up doesn't appear every time I change screen orientation?

In the past I have solved this by decompiling the app and changing the line in AndroidManifest.xml but I don't want to do this every time I update. Can't I just add a patch to to /koreader/patches/ ?
